packages/pass-style/src/passStyleOf.js

```diff
@@ -215,7 +215,8 @@ const makePassStyleOf = passStyleHelpers => {
   return harden(passStyleOf);
 };
 
-export const PassStyleOfEndowmentSymbol = Symbol.for('@endo passStyleOf');
+export const PassStyleOfEndowmentSymbol =
```

Comment:
Revert this to `Symbol.for` since it has nothing to do with passability and needs to be a registered symbol in any case.

Reply:
Done